cmd/osv-scanner/scan/source/command.go

```diff
@@ -50,14 +50,6 @@ func Command(stdout, stderr io.Writer) *cli.Command {
 				Usage: "also scan files that would be ignored by .gitignore",
 				Value: false,
 			},
-			&cli.StringSliceFlag{
```

Comment:
Can you describe in the PR description this move? E.g. enabling call analysis on all scanning types.